torch/onnx/symbolic_opset12.py

```diff
@@ -47,16 +47,26 @@ def outer(g, input, other):
     return einsum_helper(g, "i,j->ij", [input, other])
 
 
-@symbolic_helper.parse_args("v", "f", "i")
-def dropout(g, input, p, train):
+def _dropout_shared(g, input, p, train):
```

Comment:
Naming: Consider a more descriptive name for the function?

Reply:
Not sure what will be more descriptive. `_dropout_core`? `_shared_dropout_exporter`? `_dropout_common_part`? They all look similar to `_dropout_shared`...